source/_integrations/slide_local.markdown

```diff
@@ -47,6 +47,10 @@ password:
   type: string
 {% endconfiguration_basic %}
 
+## Configuration
```

Comment:
Two things:

1. This is a duplicate heading. `{% include integrations/config_flow.md %}` adds the heading "Configuration"
2. Is this meant to describe configuration options? If so, use the heading "Options" and the `{% configuration_basic %}` parameter structure.